rebar.config

```diff
@@ -43,6 +43,19 @@
              , deprecated_functions
              ]}.
 
-{project_plugins, [rebar3_hex, rebar3_lint]}.
+{project_plugins, [
+    rebar3_hex,
+    rebar3_lint,
+    rebar3_hank
+]}.
 
 {alias, [{test, [lint, xref, dialyzer, ct, cover]}]}.
+
+%% == hank ==
+
+{hank, [
+    {ignore, [
+        {"example/**", unnecessary_function_arguments},
+        {"test/**", unnecessary_function_arguments}
```

Comment:
This shouldn't be necessary. Just adding `-behavior(ct_suite).` to the test modules should be enough.

Reply:
This is under CI: that behaviour is not 21-compatible. Do you prefer a macro, there?

Reply:
Replied at https://github.com/project-fifo/rebar3_lint/pull/37#discussion_r676310071

Reply:
Pushed.